casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -36,6 +38,7 @@ trait Casper[F[_], A] {
 
 trait MultiParentCasper[F[_]] extends Casper[F, IndexedSeq[BlockMessage]] {
   def blockDag: F[BlockDag]
+  def normalizedInitialFault(weights: Map[Validator, Int]): F[Float]
```

Comment:
Could you explain the name of this method, in what sense is this number "initial"?

Reply:
As discussed earlier today, it is initial as this is the number of faults that have been accumulated so far. We want the clique oracle to give us a guarantee that is greater than this initial fault weight combined with our fault tolerance threshold t.